### DISCUSSION TITLE
Clickhouse Cloud x Azure integration roadmap ?

### DISCUSSION THREAD

**Discussion by QuentinNav** [ClickHouse/ClickHouse#50140] [General]
I know Azure integration for Clickhouse Cloud is in the books, but can you share an broad estimate as to the expected timeline of that integration ? Are we talking 2023 ?

**Comment by tbragin** [comment]
@QuentinNav Thank you for your interest! Azure is definitely next.

No certainty in timing yet. Azure is notoriously different from other cloud providers to integrate with, so I think it will take a while after GCP is announced in GA. As a result, I do not expect it to be ready this calendar year, but let me come back to this thread, when we have a better sense.

**Comment by Sebhansson** [comment]
@tbragin
 Any updates on this?

**Comment by tbragin** [comment]
@Sebhansson Thank you so much for the ping!

We have started work on the Azure POC! So far the engineering updates are encouraging, but the team has not yet shared a firm timeline (a few unknowns are still in play). Assuming we do not run into major blockers, we should see a public-facing milestone by Q1 2024 (this is based on experience with other cloud providers), and there may be an earlier point for private preview. 

We now have a waiting list for users that would like to receive updates on our progress wrt Azure here: https://clickhouse.com/cloud/azure-waitlist

**Comment by blaszta** [comment]
Found this recently, but haven't try it: https://bitnami.com/stack/clickhouse/cloud

**Comment by tomtom215** [comment]
Bumping this as we would love to see further Azure integration, fingers crossed that progress has been steady and still is roadmapped for this quarter!

**Comment by tbragin** [comment]
Tom - Yes, private preview for ClickHouse Cloud on Azure is still aimed at this quarter! Hopefully you are already on the wait list and engaged with our field representatives to get early access!

**Comment by tbragin** [comment]
I'm happy to share that ClickHous Cloud Azure support has been in Private Preview the past month or so and is now available in Public Beta as of today! https://clickhouse.com/blog/clickhouse-cloud-is-now-on-azure-in-public-beta
